web/src/routes/OrganizationsList/Organization/Tabs/Settings/GeneralSettings.tsx

```diff
@@ -0,0 +1,354 @@
+import {useEffect, useState, ReactNode} from 'react';
+import {
+  Flex,
+  FormGroup,
+  Form,
+  FormAlert,
+  TextInput,
+  FormSelect,
+  FormSelectOption,
+  ActionGroup,
+  Button,
+  Alert,
+  AlertGroup,
+  Grid,
+  GridItem,
+  FormHelperText,
+  HelperText,
+  HelperTextItem,
+} from '@patternfly/react-core';
+import {useCurrentUser} from 'src/hooks/UseCurrentUser';
+import {useOrganization} from 'src/hooks/UseOrganization';
+import {useOrganizationSettings} from 'src/hooks/UseOrganizationSettings';
+import {IOrganization} from 'src/resources/OrganizationResource';
+import {humanizeTimeForExpiry, getSeconds, isValidEmail} from 'src/libs/utils';
+import {addDisplayError} from 'src/resources/ErrorHandling';
+import {useQuayConfig} from 'src/hooks/UseQuayConfig';
+import {useUpdateUser} from 'src/hooks/UseCurrentUser';
+
+type validate = 'success' | 'warning' | 'error' | 'default';
+const timeMachineOptions = {
+  '0s': 'a few seconds',
+  '1d': 'a day',
+  '1w': '7 days',
+  '2w': '14 days',
+  '4w': 'a month',
+};
+
+type GeneralSettingsProps = {
+  organizationName: string;
+};
+
+export const GeneralSettings = (props: GeneralSettingsProps) => {
+  const quayConfig = useQuayConfig();
+  const organizationName = props.organizationName;
+  const {user, loading: isUserLoading} = useCurrentUser();
+  const {organization, isUserOrganization, loading} =
+    useOrganization(organizationName);
+  const [error, setError] = useState<string>('');
+
+  const {updateOrgSettings} = useOrganizationSettings({
+    name: organizationName,
+    onSuccess: (result) => {
+      setAlerts((prevAlerts) => {
```

Comment:
You can use `useAlerts` hook. Eg: https://github.com/quay/quay/blob/master/web/src/routes/OrganizationsList/Organization/Tabs/TeamsAndMembership/TeamsView/TeamViewKebab.tsx#L25C7-L25C15

Reply:
Similar for other places where you are setting Alerts.

Reply:
Updated to use useAlerts hook